README.md

```diff
@@ -185,20 +135,38 @@ Single tests can be executed with:
 
 	./networkit_tests --gtest_filter=CentralityGTest.testBetweennessCentrality
 
-Additionally, one can specify the level of the logs outputs by adding `--loglevel=<log_level>`;
+Additionally, one can specify the level of the logs outputs by adding `--loglevel <log_level>`;
 supported log levels are: `TRACE`, `DEBUG`, `INFO`, `WARN`, `ERROR`, and `FATAL`.
 
+
 ### Compiling with address/leak sanitizers
 
-Address and leak sanitizers are very useful tools to test if your code produces memory errors/leaks.
-To compile your code with sanitizers, set the [CMake] `NETWORKIT_WITH_SANITIZERS` to either `address` or `leak`:
+Sanitizers are great tools to debug your code. NetworKit provides additional [Cmake] flags
+to enable address, leak, and undefined behavior sanitizers.
+To compile your code with sanitizers, set the [CMake]
+`NETWORKIT_WITH_SANITIZERS` to either `address` or `leak`:
 
-	cmake -DNETWORKIT_WITH_SANITIZERS='leak'
+	cmake -DNETWORKIT_WITH_SANITIZERS=leak
 
 By setting this flag to `address`, your code will be compiled with the `address` and the `undefined` sanitizers.
 Setting it to `leak` also adds the `leak` sanitizer.
 
-## Known Issues
+
+# Documentation
+
+The most recent version of the [documentation can be found online](https://networkit.github.io/dev-docs/index.html).
+
+# Contact
+
+For questions regarding NetworKit, subscribe to our [e-mail list][list] and feel free to ask.
+
+# Known Issues
```

Comment:
I am not sure if we want to keep this. The GitHub issue tracker seems to be a much better tool to search for known issues.